Test/Core/ExecuteTableTests.cs

```diff
@@ -180,6 +180,41 @@ public void ExecuteTableWithAnonymousConverter_Test()
             }
         }
 
+        [TestMethod]
+        public void ExecuteTableWithMapTo_Test()
+        {
+            using (var cmd = new DatabaseCommand(_connection))
+            {
+                cmd.Log = Console.WriteLine;
+                cmd.CommandText = @" SELECT EMPNO,
+                                            ENAME,
+                                            SAL,
+                                            HIREDATE,
+                                            COMM,
+                                            MGR,
+                                            DEPT.DNAME
+                                       FROM EMP
+                                  LEFT JOIN DEPT ON DEPT.DEPTNO = EMP.DEPTNO";
+                var employees = cmd.ExecuteTable((row) =>
+                {
+                    var emp = row.MapTo<MyEmployee>();
+                    emp.Department = row.MapTo<MyDepartment>();
+                    return emp;
+                });
+
+                var smith = employees.First();
+
+                Assert.AreEqual(14, employees.Count());
+                Assert.AreEqual(EMP.Smith.EmpNo, smith.EmpNo);
```

Comment:
It's better to set the expected value with associated string (ex. "SMITH", 7336, ...)